source/ftps4.c

```diff
@@ -272,7 +272,7 @@ static char file_type_char(mode_t mode)
 		S_ISLNK(mode) ? 'l' : ' ';
 }
 
-static int gen_list_format(char *out, int n, mode_t mode, unsigned int file_size,
+static int gen_list_format(char *out, int n, mode_t mode, long long int file_size,
 	int month_n, int day_n, int hour, int minute, const char *filename)
```

Comment:
Should be `unsigned long long` or better, `uint64_t`. I doubt file size can go < 0